frontend/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
```

Comment:
are we using emotion styling? or is that a dependency for mui or something?

Reply:
huh, interesting to find out, emotion styling is pretty pog in general imo and i use it for most of my stuff.